python/cuml/fil/fil.pyx

```diff
@@ -578,9 +634,36 @@ class ForestInference(Base):
     def load_from_randomforest(self,
                                model_handle,
                                output_class=False,
-                               algo='AUTO',
-                               storage_type='DENSE',
+                               algo='auto',
+                               storage_type='auto',
                                threshold=0.50):
+        """
+        Returns a FIL instance by converting cuML's random forest model to
```

Comment:
Does this convert from random forest or from treelite? I know we use it for RF, but structurally it seems like this actually takes a treelite forest model